plugins/bnr-dp/fu-bnr-dp-device.c

```diff
@@ -742,6 +773,29 @@ fu_bnr_dp_device_write_firmware(FuDevice *device,
 
 	/* give controller some time before ->reload() tries to read info again */
 	fu_device_sleep(device, 3000);
+
+	st_info_flags_post =
+	    fu_bnr_dp_device_info_flags(self, FU_BNR_DP_MODULE_NUMBER_RECEIVER, error);
+	if (st_info_flags_post == NULL) {
+		g_prefix_error_literal(error,
+				       "failed to read device info flags after writing firmware");
```

Comment:
the `g_prefix_error_literal` functions just prefix in a blind way, so you need to do something like `"failed to read device info flags after writing firmware: "` with the suffix.